lib/ansible/module_utils/storage/hpe3par/hpe3par.py

```diff
@@ -0,0 +1,90 @@
+# Copyright: (c) 2018, Hewlett Packard Enterprise Development LP
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from ansible.module_utils import basic
+
+
+def convert_to_binary_multiple(size_with_unit):
+    if size_with_unit is None:
+        return -1
+    valid_units = ['MiB', 'GiB', 'TiB']
+    valid_unit = False
+    for unit in valid_units:
+        if size_with_unit.strip().endswith(unit):
+            valid_unit = True
+            size = size_with_unit.split(unit)[0]
+            if float(size) < 0:
+                return -1
+    if not valid_unit:
+        raise ValueError("%s does not have a valid unit. The unit must be one of %s" % (size_with_unit, valid_units))
+
+    size = size_with_unit.replace(" ", "").split('iB')[0]
+    size_kib = basic.human_to_bytes(size)
+    return int(size_kib / (1024 * 1024))
+
+
+storage_system_spec = {
```

Comment:
Appologies I missed this when reviewing. The common arguments must be moved to `module_utils/storage/hpe3par/`.

Reply:
That's already the case, no ?
![screenshot from 2019-02-26 15-41-20](https://user-images.githubusercontent.com/388198/53420933-0242d200-39dd-11e9-947a-bd6d6619a599.png)

Reply:
ah, looks like you are right https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/storage/hpe3par/hpe3par.py#L26-L45

@farhan7500 In a follow up PR can you please remove the duplicated options.